modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm

```diff
@@ -384,6 +384,11 @@ GLOBAL_LIST_EMPTY(customizable_races)
 						eye_emissive.pixel_y += species_human.dna.species.offset_features[OFFSET_FACE][2]
 						standing += eye_emissive
 
+		// blush
+		if (HAS_TRAIT(species_human, TRAIT_BLUSHING)) // Caused by either the *blush emote or the "drunk" mood event
+			var/mutable_appearance/blush_overlay = mutable_appearance('icons/mob/human_face.dmi', "blush", -BODY_ADJ_LAYER) //should appear behind the eyes
+			blush_overlay.color = COLOR_BLUSH_PINK
+			standing += blush_overlay
```

Comment:
How does this decide whether or not you're having those visuals on your character? Is it handled by the emote itself not giving you the TRAIT_BLUSHING if you haven't got the pref on? I just want to be sure this isn't forced on everyone if they have the pref for it off.

Reply:
Doesn't have a pref, it's a default /tg/ behavior https://github.com/tgstation/tgstation/pull/66321